### ISSUE TITLE
https://rsa8192.badssl.com/ cert has expired

### ISSUE THREAD

**Issue by inikulin** [chromium/badssl.com#499]
<img width="1137" alt="image" src="https://user-images.githubusercontent.com/453071/160433527-f7af9940-4f46-4cea-ab54-2e9444e37054.png">

**Comment by christhompson** [comment]
Fixed in d084af689d92e91ee540d6930374a44997da7af3. The new cert should be valid through 2023/03/28.

**Comment by cpu** [comment]
@christhompson Happy anniversary to this issue :laughing: It seems like this has happened again. Would you be able to re-issue the `rsa8192.badssl.com` cert once more?

```
$ openssl s_client -connect rsa8192.badssl.com:443 </dev/null 2>/dev/null | openssl x509 -noout -enddate
notAfter=Mar 27 23:59:59 2024 GMT
```
